package.yaml

```diff
@@ -65,8 +65,10 @@ tests:
     - cardano-crypto
     - cardano-crypto-wrapper
     - cardano-ledger
+    - cardano-ledger-test
     - cardano-transactions
     - cborg
+    - hedgehog-quickcheck
```

Comment:
`QuickCheck` is not good anymore ? ;-)

Reply:
It is! But most of the generator in `cardano-ledger` are using Hedgehog generators! This `hedgehog-quickcheck` library allows for transforming a `Hedgehog` generator into a `QuickCheck` one